tutum.yml

```diff
@@ -1,11 +1,11 @@
 master:
-  image: 'citusdata/citus:5.2.2'
```

Comment:
This shouldn't be updated any longer; we leave it here as legacy but since we switched away from a workerlist file it hasn't worked, so it's stuck with Citus 5.